src/Microsoft.Data/PrimitiveColumn.DataFrameFunctions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class PrimitiveColumn<T> : BaseColumn
+    where T : unmanaged
+    {
+        public override BaseColumn Clip<U>(U lower, U upper)
+        {
+            object convertedLower = Convert.ChangeType(lower, typeof(T));
+            if (typeof(T) == typeof(U) || convertedLower != null)
+            {
+                return _Clip((T)convertedLower, (T)Convert.ChangeType(upper, typeof(T)));
+            }
+            else
+                throw new ArgumentException(Strings.MismatchedValueType + typeof(T).ToString(), nameof(U));
+        }
+
+        public override DataFrame Description()
+        {
+            DataFrame ret = new DataFrame();
+            StringColumn stringColumn = new StringColumn("Description", 0);
+            stringColumn.Append("Length");
+            stringColumn.Append("Max");
+            stringColumn.Append("Min");
+            stringColumn.Append("Mean");
+            float max = (float)Convert.ChangeType(Max(), typeof(float));
```

Comment:
is using `float` an issue when the column is of type `double`?

Reply:
Not that I have seen yet. We will lose some accuracy sure, but I figured it wouldn't matter much for a `Description` method.